index.html

```diff
@@ -265,64 +300,68 @@ <h1 class="project-heading">Multi-Post Stories</h1>
-                <img
-                  src="./Assets/Icons/medium-vector.svg"
-                  alt="medium"
-                  class="social-icons"
-                />
-              </a>
-            </li>
-            <li>
-              <a href="#">
-                <img
-                  src="./Assets/Icons/github-vector.svg"
-                  alt="gitHub"
-                  class="social-icons"
-                />
-              </a>
-            </li>
-            <li>
-              <a href="#">
-                <img
-                  src="./Assets/Icons/social-icon-vector.svg"
-                  alt="social"
-                  class="social-icons"
-                />
-              </a>
-            </li>
-          </ul>
-          <button class="btn project-link" type="button">Get my resume</button>
-          <div class="">
+        <div class="about-container">
+          <div class="about">
+            <h1 class="about-header">About Myself</h1>
+            <h1 class="about-text">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </h1>
+            <p class="connect">LET’S CONNECT</p>
+            <ul class="connect-links">
+              <li>
+                <a href="#">
+                  <img
+                    src="./Assets/Icons/twitter-vector.svg"
+                    alt="twitter"
+                    class="social-icons"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    src="./Assets/Icons/linkedin-vector.svg"
+                    alt="linkedin"
+                    class="social-icons"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    src="./Assets/Icons/medium-vector.svg"
+                    alt="medium"
+                    class="social-icons"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    src="./Assets/Icons/github-vector.svg"
+                    alt="gitHub"
+                    class="social-icons"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    src="./Assets/Icons/social-icon-vector.svg"
+                    alt="social"
+                    class="social-icons"
+                  />
+                </a>
+              </li>
+            </ul>
+            <button class="btn project-link" type="button">
+              Get my resume
+            </button>
+          </div>
+          <div class="language-container">
```

Comment:
- [ ] Kindly check on your framework and skills, there is a down-pointing symbol missing;

<img width="311" alt="image" src="https://user-images.githubusercontent.com/61408860/171621021-f5902495-e759-4c5e-a928-3d90d3541cdc.png">